impress-org/givewp-next-gen
# Feature: add donation confirmation settings UI

## Description



This updates the donation confirmation settings UI with a popover and list of available template tags (pulled from v2 forms).

## Affects



The donation confirmation settings UI in the form builder.  Also the confirmation page now uses Interweave to display the rich text from the template tags that were generated.

## Visuals



<img width="743" alt="Screenshot 2023-08-03 at 12 40 15 PM" src="https://github.com/impress-org/givewp-next-gen/assets/10138447/5f6f3551-38e8-4b7d-9a02-6bc0d776e4e3">

## Testing Instructions



- Build assets and view a donation form builder
- Interact with the donation confirmation settings
- Make sure v2 form template tags work as expected

## Pre-review Checklist






---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1203889335107900
